client/state/rewind/preflight/hooks.tsx

```diff
@@ -29,8 +31,8 @@ export const usePreflightStatusQuery = ( siteId: number ): UseQueryResult => {
 				apiNamespace: 'wpcom/v2',
 			} );
 		},
-		enabled: shouldFetch(),
-		refetchInterval: 5000,
+		enabled: shouldFetch,
+		refetchInterval: 3000,
```

Comment:
Anything that led us to 3s from 5s?  Just curious if there was a technical reason or more of "it felt too slow"

Reply:
I was messing around with my dev environment and forgot to keep it as it was. I think 5s still good. I just switch it back.